packages/kbn-utils/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@kbn/utils",
+  "main": "./target/index.js",
+  "version": "1.0.0",
+  "license": "Apache-2.0",
+  "private": true,
+  "scripts": {
+    "build": "tsc",
+    "kbn:bootstrap": "yarn build",
+    "kbn:watch": "yarn build --watch"
+  },
+  "dependencies": {
+    "@kbn/config-schema": "1.0.0"
+  },
+  "devDependencies": {
+    "typescript": "3.9.5"
```

Comment:
why not `4.0.2` as master?

Reply:
Was pulled from another PR which was out-of-date. Fixing.

Reply:
https://github.com/elastic/kibana/pull/76518/commits/c9eef0eb0293042b98a562a37cb8d47a519a4b1c#diff-ad5b156108588b5f388e8cf1ea7f4810R16